docs/sdks/server-sdks/python.md

```diff
@@ -50,9 +50,14 @@ else:
 * `<SUBJECT-KEY>` is the value that identifies each entity in your experiment, typically `user_id`;
 * `<FLAG-KEY>` is the key that you chose when creating a flag; you can find it on the [flag page](https://eppo.cloud/feature-flags). For the rest of this presentation, we’ll use `"test-checkout"`. To follow along, we recommend that you create a test flag in your account, and split users between `"fast_checkout"` and `"standard_checkout"`.
```

Comment:
Do we need to show people where to find an API Token/Flag key?

Reply:
> Do we need to show people where to find an API Token/Flag key?

Are you suggesting adding something else besides [this](https://github.com/Eppo-exp/eppo-docs/pull/299/files#diff-281041555d0a3483d99a53573bbb9849f2a8704a70eb1bbb4f9b12ab72f86b95R23)?

Reply:
I was curious if you felt that was enough.